simapp/v2/app_di.go

```diff
@@ -116,7 +116,7 @@ func NewSimApp(
 					RootDir: homeDir,
 					SSType:  0,
```

Comment:
we need to make sure these can be modified from the config.toml. I also question if these should even be in app.go? cc @julienrbrt

Reply:
talked with julien, he mentioned this was done due to config not being ready but now it is. we can merge this then abstract it out